include/cartesian_geom/cartesian_kernel.h

```diff
@@ -0,0 +1,20 @@
+#include "point.h"
```

Comment:
You can put a copyright to all the source files of the library like 
```
// VolEsti (volume computation and sampling library)

// Copyright (c) 20012-2018 Vissarion Fisikopoulos
// Copyright (c) 2018 Apostolos Chalkis

//You may redistribute or modify the software under the GNU Lesser General Public License as published by Free Software Foundation, either version 3 of the License, or (at your option) any later version. It is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY.
```

After copyright you can enter the following line to ack gsoc in all the files you contribute during gsoc

//Contributed and/or modified by Apostolos Chalkis, as part of Google Summer of Code 2018 program.